libmscore/page.cpp

```diff
@@ -399,9 +407,11 @@ void PageFormat::read(XmlReader& e, Score* score)
                         }
                   }
             else if (tag == "page-height")
-                  _size.rheight() = e.readDouble() * 0.5 / PPI;
+                  _size.rheight() = e.readDouble() / SCALE_XML;
             else if (tag == "page-width")
-                  _size.rwidth() = e.readDouble() * .5 / PPI;
+                  _size.rwidth() = e.readDouble() / SCALE_XML;
+            else if (tag == "page-units")
```

Comment:
OK, I'll make that change.  I was also confused by the original line note, the selected line should probably have been one line down, the one below this note.
I can see how storing the string is more human-readable within the file, more xml-like.
This is entangled with the unitSuffixes[] discussion elsewhere, and I'm still wondering about the translations, and if they need to be integrated into the Page Settings dialog, and if that has any ripple effects here.  I'll get this going and see how that other discussion evolves...